newscoop/library/Newscoop/Installer/Services/UpgradeService.php

```diff
@@ -164,4 +163,23 @@ public function upgradeDatabase($versionsArray, $silent = false, $showRolls = fa
 
         return true;
     }
+
+    /**
+     * Gets default alias for the first publication
+     *
+     * @return string alias
+     */
+    public function getDefaultAlias()
```

Comment:
well, we need only default alias and that's all, dont think so we need to enable api for other since they wont be used in upgrade script anyhow